src/validation/custom.validation.ts

```diff
@@ -295,3 +295,35 @@ const checkCorporatesAddress = (trust: trustType.Trust, addressMaxLength: number
     }
   }
 };
+
+export const validateEmail = (email: string, maxLength: number) => {
+  const isEmailPresent = checkEmailIsPresent(email);
+  if (isEmailPresent) {
+    const isWithinLengthLimit = checkIsWithinLengthLimit(email, maxLength);
+    if (isWithinLengthLimit) {
+      checkCorrectIsFormat(email);
+    }
+  }
+  return true;
+};
+
+const checkEmailIsPresent = (email: string) => {
+  if (email === undefined || email === "") {
```

Comment:
Probably email needs to be trimmed before checking it.

Reply:
Trim the email as it is passed in.